src/pages/User/User.js

```diff
@@ -48,65 +49,57 @@ class User extends React.Component {
 
   render() {
     const { data, username, isLoading } = this.props.user;
-    const { value } = this.state;
-    const isPhotos = value === 0;
-    const isCollections = value === 1;
-    const isStats = value === 2;
     const readyToLoad = data && !isLoading;
+
+    const convertedNumbers = (x) => {
+      return x.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ",");
+    };
```

Comment:
I would take this function from render and put it outside the class, on line 22 for example, there is no need to create this function in each render